keyboards/keebio/quefrency/keymaps/rogthefrog/keymap.c

```diff
@@ -0,0 +1,31 @@
+#include QMK_KEYBOARD_H
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Reply:
Happy to! 

The other keymap.c files for that keyboard don't have that header. The .h files do. Should I still add it to my keymap.c?

Reply:
That's fine.  We haven't gone through an done that, partly because it's a LOT of work.  So we're just making sure that any new or modified files have the license headers

Reply:
Added